frontend/src/pages/Register.vue

```diff
@@ -105,6 +105,7 @@
                       </b-form-checkbox>
                     </b-col>
                   </b-row>
+                  <!-- Wolle: remove this? or shall the alert or a toaster be shown? -->
                   <b-alert
                     v-if="showError"
                     show
```

Comment:
Remove this? @Mogge @ogerly 
It seems not functioning anyways, to me …

Or shall the alert be shown?

Reply:
removed after talk to @ogerly